scripts/lib/CIME/code_checker.py

```diff
@@ -75,9 +75,12 @@ def get_all_checkable_files():
 ###############################################################################
     cimeroot = get_cime_root()
     all_git_files = run_cmd_no_fail("git ls-files", from_dir=cimeroot, verbose=False).splitlines()
-
+    if get_cime_default_driver() == "nuopc":
```

Comment:
Curious as to why this was needed. all_git_files are computed from cimeroot, so wouldn't that include the nuopc subdirectory?

Reply:
Because the nuopc subdirectory is no longer part of the same git repo.

Reply:
Ah, ok.